cmd/crossplane/core/core.go

```diff
@@ -85,9 +85,10 @@ type startCommand struct {
 
 	EnableCompositionRevisions bool `group:"Beta Features:" help:"Enable support for CompositionRevisions." default:"true"`
 
-	EnableEnvironmentConfigs   bool `group:"Alpha Features:" help:"Enable support for EnvironmentConfigs."`
-	EnableExternalSecretStores bool `group:"Alpha Features:" help:"Enable support for External Secret Stores."`
-	EnableCompositionFunctions bool `group:"Alpha Features:" help:"Enable support for Composition Functions."`
+	EnableEnvironmentConfigs                 bool `group:"Alpha Features:" help:"Enable support for EnvironmentConfigs."`
+	EnableExternalSecretStores               bool `group:"Alpha Features:" help:"Enable support for External Secret Stores."`
+	EnableCompositionFunctions               bool `group:"Alpha Features:" help:"Enable support for Composition Functions."`
+	EnableCompositionWebhookSchemaValidation bool `group:"Alpha Features:" help:"Enable Composition validation using schemas."`
```

Comment:
I was thinking to keep it behind a single flag, in case we can always further specify it:
```go
	EnableCompositionWebhookSchemaRenderedValidation bool `group:"Alpha Features:" help:"Enable Composition validation for rendered resources using schemas."`
```
Or something along these lines.

Reply:
I would vote for a single flag for both phase 2 & phase 3 for brevity. I cannot come up with a much better one than `EnableCompositionWebhookSchemaValidation`, so I am good as is as well. Just wanted to confirm.